cloud-regionsrv-client.spec

```diff
@@ -210,6 +210,15 @@ gzip %{buildroot}/%{_mandir}/man1/*
 %pre addon-azure
 %service_add_pre regionsrv-enabler.timer
 
+%preun
+# Do not run during an upgrade process
+# WHen the package is removed we need to clean up or we will leave
+# repositories with "plugin://" behind while the plugin we supply is
+# being removed (bsc#1240310)
+if [ $1 -eq 0 ]; then
```

Comment:
-- SC2086 (info): Double quote to prevent globbing and word splitting.

```
if [ "$1" -eq 0 ]; then
```

Reply:
Thanks fixed